src/lib.js

```diff
@@ -93,6 +93,14 @@ export {
   CommentHeader,
   CommentActions
 } from './components/Comment'
+export {
```

Comment:
IMO skip exposing individual elements unless truly needed. This way we can refactor how the schema is implemented without breaking a «public» API.

Reply:
I agree, but I'm using them all on the cheat sheet page.